3lvia/kunde-elvia-web
# Fix/komega 1121 missing active link style in main menu

OMEGA-46

## What is the reason for this change?
Previously, users had difficulties to see where they currently was in the main menu.

### What is the expected result?
With this change, the active main menu button receives an active style.

## How can it be tested and verified?
Check that the implementation complies with the designs in [Figma](https://www.figma.com/file/sJHoScmQRq0hlr6AcHI9WB/Elvia.no?node-id=9732%3A226), and that it complies with the description in the [Jira issue](https://elvia.atlassian.net/browse/OMEGA-46).
